sssd_test_framework/hosts/base.py

```diff
@@ -134,6 +134,12 @@ def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
         self.client: dict[str, Any] = self.config.get("client", {})
 
+        self.domain_str: str = self.config.get("domain", "test")
```

Comment:
Is it really a domain of directory server or a Kerberos domain?

Reply:
It's the domain. 

https://github.com/SSSD/sssd/pull/7156/files